v3/kafka.go

```diff
@@ -126,6 +149,15 @@ func setConfig(configList []*BrokerConfig, brokers []string) (BrokerConfig, erro
 		config.BaseWriterConfig.WriteTimeout = config.WriteTimeout
 	}
 
+	if config.BaseWriterConfig.BatchTimeout == 0 {
+		config.BaseWriterConfig.BatchTimeout = writerDefaultBatchTimeout
+	}
+
+	if config.BaseWriterConfig.RequiredAcks == 1 {
```

Comment:
since we changed how we initiate the kafka writer by using `&kafka.Writer` directly, it's now actually possible to use `RequiredAcks = 0`, which means there are 3 possible values. We need to adjust this warning to `if RequiredAcks != -1`